data/maps.tsx

```diff
@@ -723,6 +765,12 @@ const skillIndexMap = {
     7: "trapping",
     8: "construction",
     9: "worship",
+    10: "cooking",
```

Comment:
https://github.com/Sludging/idleon-efficiency/blob/main/data/domain/SkillsIndex.tsx this is the more appropriate skills index to use.

Again sorry for having this legacy code around, I know it's confusing .. I'm hoping to get rid of it soon.

Reply:
Yeah don't worry, just completed the list in case it was still used somewhere, I'm using the SkillsIndex when in need